sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -206,11 +238,8 @@ private[ui] abstract class ExecutionTable(
   }
 
   def toNodeSeq(request: HttpServletRequest): Seq[Node] = {
-    <div>
-      <h4>{tableName}</h4>
```

Comment:
It's just a detail, but I was unclear why you don't need the table name anymore. It's because all other code paths already render the table name, and `UIUtils.listingTable` does the rest?

Reply:
> It's just a detail, but I was unclear why you don't need the table name anymore.

Table name for running executions was 'Running Queries ({running.size})', for  completed executions was 'Completed Queries ({completed.size})' and for failed executions was 'Failed Queries ({failed.size})'.

This is already there in the code, inside if condition. 
 ```
          <h4>
              <span class="collapse-table-arrow arrow-open"></span>
              <a>Running Queries ({running.size})</a>
            </h4>
```

So, the table name is no longer required in this case.